index.html

```diff
@@ -14,11 +14,16 @@
         <a href="#" class="logo">My Logo</a>
         <nav>
             <button type="button" class="hamburger">&#9776;</button>
```

Comment:
Kinldy note that the hamburger button goes out of the view on mobile screen 🤗
Also please remove the spacing around the whole website, it should get removed in mobile view when you remove it in the Desktop design. Also maybe round the bottom left corner as shown in the Figma design 😃

Your design:

![image](https://user-images.githubusercontent.com/73714615/188266186-f870acc4-a3d2-4419-a2f4-2d9f8f1a91d8.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188266198-5ddd04c3-16d0-4074-afd8-4caf5fca7c6a.png)

Reply:
Kindly, I recommend you follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉